installinator/src/hardware.rs

```diff
@@ -22,9 +22,10 @@ impl Hardware {
             .context("failed to detect whether host is a gimlet")?;
         ensure!(is_gimlet, "hardware scan only supported on gimlets");
 
-        let hardware = HardwareManager::new(log, None).map_err(|err| {
-            anyhow!("failed to create HardwareManager: {err}")
-        })?;
+        let hardware =
```

Comment:
Is this change necessary? Not a huge deal, but checking I'm not missing anything.

Reply:
There seems to have been a deviation of sorts early in the development of this PR / branch. I think the original branch I started working from used `Logger` as the parameter, meanwhile as `main` progressed, it might have been changed to `&Logger`.